src/sources/soundsourcesndfile.cpp

```diff
@@ -47,17 +46,17 @@ QStringList getSupportedFileExtensionsFiltered() {
 const QString SoundSourceProviderSndFile::kDisplayName = QStringLiteral("libsndfile");
 
 SoundSourceProviderSndFile::SoundSourceProviderSndFile()
-        : m_supportedFileExtensions(getSupportedFileExtensionsFiltered()) {
+        : m_supportedFileTypes(getSupportedFileTypesFiltered()) {
 }
 
-QStringList SoundSourceProviderSndFile::getSupportedFileExtensions() const {
-    return m_supportedFileExtensions;
+QStringList SoundSourceProviderSndFile::getSupportedFileTypes() const {
+    return m_supportedFileTypes;
 }
 
 SoundSourceProviderPriority SoundSourceProviderSndFile::getPriorityHint(
-        const QString& supportedFileExtension) const {
-    if (supportedFileExtension.startsWith(QStringLiteral("aif")) ||
-            supportedFileExtension == QLatin1String("wav")) {
+        const QString& supportedFileType) const {
+    if (supportedFileType.startsWith(QStringLiteral("aif")) ||
```

Comment:
Already inconsistent in existing code: We should use either `QStringLiteral` or `QLatin1String`. Preferably the latter if it suffices.